cloudwatch-ingester-axiom-cloudformation-stack.template.yaml

```
@@ -33,36 +35,36 @@ Parameters:
     Default: false
     Description: Whether or not disable json logs. defaults to false.
 Conditions:
-  HasCloudWatchLogGroupName: !Not
+  HasCloudWatchLogGroupNames: !Not
     - !Equals
-      - ''
-      - !Ref CloudWatchLogGroupName
+      - !Join ["", !Ref CloudWatchLogGroupNames]
+      - ""
 Resources:
-  LogGroupSubscriptionFilter:
-    Type: AWS::Logs::SubscriptionFilter
-    Condition: HasCloudWatchLogGroupName
-    Properties:
-      DestinationArn: !GetAtt
-        - LogsLambda
-        - Arn
-      FilterPattern: ""
-      LogGroupName: !If
-        - HasCloudWatchLogGroupName
-        - !Ref CloudWatchLogGroupName
-        - !Ref AWS::NoValue
+  "Fn::ForEach::SubscriptionFilters":
+    - groupName
+    - !Ref CloudWatchLogGroupNames
+    - "lgsf&{groupName}":
+        Type: AWS::Logs::SubscriptionFilter
+        Condition: HasCloudWatchLogGroupNames
+        Properties:
+          DestinationArn: !GetAtt
+            - LogsLambda
+            - Arn
+          FilterPattern: ""
+          LogGroupName: !Ref groupName
```

```suggestion
    - GroupName
    - !Ref CloudWatchLogGroupNames
    - "LGSF&{GroupName}":
        Type: AWS::Logs::SubscriptionFilter
        Condition: HasCloudWatchLogGroupNames
        Properties:
          DestinationArn: !GetAtt
            - LogsLambda
            - Arn
          FilterPattern: ""
          LogGroupName: !Ref GroupName
```
I think the standard here is TitleCase